src/extraction/FStar.Extraction.ML.Term.fs

```diff
@@ -1287,6 +1295,8 @@ and term_as_mlexpr' (g:uenv) (top:term) : (mlexpr * e_tag * mlty) =
           begin
                match try_lookup_fv g fv with
                | None -> //it's been erased
+                 // Errors.log_issue t.pos (Errors.Error_CallToErased,
```

Comment:
Did we settle on how to actually make this work properly? (Some discussion in #2090)

Reply:
Yes, I think this is fixed properly in guido_noextract. Thanks.